envbuilder.go

```diff
@@ -156,6 +157,34 @@ func Run(ctx context.Context, options Options) error {
 		}
 	}
 
+	gitURLParsed, err := url.Parse(options.GitURL)
+	if err != nil {
+		return fmt.Errorf("invalid git URL: %w", err)
+	}
+	// If we're cloning over SSH, we need a known_hosts file.
+	if gitURLParsed.Scheme == "ssh" {
+		var knownHostsContent []byte
+		if options.GitSSHKnownHostsBase64 != "" {
+			if kh, err := base64.StdEncoding.DecodeString(options.GitSSHKnownHostsBase64); err != nil {
+				return fmt.Errorf("invalid known_hosts content: %w", err)
```

Comment:
Would make sense to make it more explicit like `invalid Base64 content`? As a user, I would think my `know_hosts` is using some wrong URL or maybe there is a typo error in the config.

Reply:
Makes sense!